aleph-client/src/lib.rs

```diff
@@ -26,10 +26,10 @@ pub use session::{
 pub use staking::{
     batch_bond as staking_batch_bond, batch_nominate as staking_batch_nominate,
     bond as staking_bond, bonded as staking_bonded, force_new_era as staking_force_new_era,
-    ledger as staking_ledger, multi_bond as staking_multi_bond, nominate as staking_nominate,
-    payout_stakers, payout_stakers_and_assert_locked_balance,
-    set_staking_limits as staking_set_staking_limits, validate as staking_validate,
-    wait_for_full_era_completion, wait_for_next_era, StakingLedger,
+    get_current_era, get_payout_for_era, ledger as staking_ledger,
```

Comment:
Why do we reexport prefixed names like that? Wouldn't it suffice to just use `staking::set_staking_limits` instead of `staking_set_staking_limits`?

Reply:
staking is a private mod